src/components/LocationCard/LocationCard.js

```diff
@@ -13,6 +13,7 @@ export const LocationCard = (props) => {
         As,
         locationId,
         locationTitle,
+        hasRadiusSlider,
         distanceRadius,
         sliderLabel,
```

Comment:
There is no way to activate the location modal window using just a keyboard. I can select the input by pressing the tab button, but I can't open the modal window.